bastet_js/src/JsArray.re

```diff
@@ -34,17 +34,13 @@ module Alt: ALT with type t('a) = array('a) = {
 module Apply: APPLY with type t('a) = array('a) = {
   include Functor;
   let apply = (fn_array, a) =>
-    Js.Array.reduce(
-      (acc, f) => Js.Array.concat(acc, map(f, a)),
-      [||],
-      fn_array,
-    );
+    Js.Array.reduce((acc, f) => Alt.alt(acc, map(f, a)), [||], fn_array);
```

Comment:
Could you do the same for `bastet/src/Array.re` and have it reference `Alt.alt`?